Remotion/Web/Core/UI/Controls/IFallbackNavigationUrlProvider.cs

```diff
@@ -0,0 +1,31 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+namespace Remotion.Web.UI.Controls
+{
+  /// <summary>
+  /// Provides the fallback URL for clickable elements that should not be opened in a new tab
+  /// </summary>
+  /// <threadsafety static="true" instance="true" />
+  public interface IFallbackNavigationUrlProvider
```

Comment:
Please add unit tests for the serivce registration and the implementation. 

As a convention, we also add a `<seealso>` to the interface and reference the implementation. it helps with the docs.